doc/tutorial/network_ring.rst

```diff
@@ -114,8 +114,12 @@ The execution
 
 To create a simulation, we must create an :class:`arbor.context` and :py:class:`arbor.domain_decomposition`.
 
-Step **(12)** creates a default execution context, and uses the :func:`arbor.partition_load_balance` to create a
-default domain decomposition. You can print the objects to see what defaults they produce on your system.
+Step **(12)** initalizes the ``threads`` parameter of :class:`arbor.context` with the ``avail_threads`` flag. By supplying
+this flag, a context is constructed that will use all locally available threads. On your local machine this will match the
+number of cpu-cores, times two if your cpu provides some form of simultaneous multithreading. Especially with large numbers
```

Comment:
`times two` .... That wholly depends on the core arch. 
Better (more accurate)
> CPU cores times the number of SMT, as supported by your CPU. :footnote: Most Intel/AMD Chips have SMT=2 aka Hyperthreading. 

SMT should be written out once.

Reply:
There are no consumer cpus with more than two, right?

Also, SMT is written out (it is never abbreviated).

Reply:
Depending on how wide you cast the (you _can_ technically buy a POWER9 workstation) there's SMT=8 consumer machines. Just say it's likely to be two, then no one will be surprised if it's not

SMT: Yes, but not in my example.

Reply:
POWER9 workstations are not mainstream!

I've added a likely, and explained in the MPI version that it is and should be not used.

Reply:
Technically, you can order one now https://www.raptorcs.com/TALOSII/